cv-pinhole/src/radial.rs

```diff
@@ -0,0 +1,161 @@
+//! Radial distortion model
+
+use num_traits::Float;
```

Comment:
To get around the check failure on this line, just use `Float::method(f)`. Basically, use the `Float` trait method using the static method syntax instead of the `f.method()` syntax, which will prevent any unused import errors. This happens because when `std` is enabled it uses those primitive methods rather than the ones on `Float`.